code/modules/surgery/organs/heart.dm

```diff
@@ -147,16 +147,23 @@
 
 /obj/item/organ/heart/cybernetic
 	name = "cybernetic heart"
-	desc = "An electronic device designed to mimic the functions of an organic human heart. Offers no benefit over an organic heart other than being easy to make."
+	desc = "An electronic device designed to mimic the functions of an organic human heart. Also holds an emergency dose of epinephrine, used automatically after facing severe trauma."
 	icon_state = "heart-c"
 	synthetic = TRUE
+	var/crituse = FALSE // doses with epi if in crit once.
 
 /obj/item/organ/heart/cybernetic/emp_act()
 	. = ..()
 	if(. & EMP_PROTECT_SELF)
 		return
 	Stop()
 
+/obj/item/organ/heart/cybernetic/on_life()
+	. = ..()
+	if(!crituse && owner.stat == UNCONSCIOUS)
```

Comment:
Won't this fire as soon as it's replaced ?

Reply:
UNCONSCIOUS stat is crit only, if i'm not wrong, so there would still be a window to replace the heart before then, with epinephrine.
Plus, you could just use corazone during the operation.

Reply:
Let me double check but I think unconscious is hard-crit

Reply:
Also losing your heart doesn't immediately put you in unconscious for true omegalols